### PR TITLE
Add light as juno token

### PR DESCRIPTION
**Pull request by staking-hub**

### CODE REVIEW

File: lumenx/assetlist.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "$schema":"../assetlist.schema.json",
-    "chain_name":"lumenx",
+    "chain_name":"lumen",
```
Comment (JeremyParish69) [inline lumenx/assetlist.json:3]: Please don't change this chain_name (which must match the folder name).

File: juno/assetlist.json [hunk 2]
```diff
@@ -1242,6 +1242,27 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/spacer.png"
       }            
     },
+    {
+      "description":"Light: LumenX community DAO treasury token",
+      "type_asset": "cw20",
```
Comment (JeremyParish69) [inline juno/assetlist.json:1247]: since this is a cw20, please include the address property

File: juno/assetlist.json [hunk 3]
```diff
@@ -1242,6 +1242,27 @@
...
+          "denom_units":[
+             {
+                "denom":"nlight",
```
Comment (JeremyParish69) [inline juno/assetlist.json:1250]: you're free to add 'nlight' as an alias of the base, but the base denom will be something like 'cw20:juno1a2b3....'

**Review by staking-hub — COMMENTED** [review]
Please ignore the PR concerning lumenx.

File: juno/assetlist.json [hunk 4]
```diff
@@ -1242,6 +1242,28 @@
...
+             {
+                "denom":"cw20:juno1dpany8c0lj526lsa02sldv7shzvnw5dt5ues72rk35hd69rrydxqeraz8l",
+                "exponent":0
+             },
+             {
+                "denom":"light",
+                "exponent":9
+             }
+          ],
+       "base":"nlight",
```
Comment (JeremyParish69) [inline juno/assetlist.json:1259]: Thanks for making the changes, although this 'base' property must exactly refer to a defined denom_unit, and therefore should also be "cw20:juno1dpany8c0lj526lsa02sldv7shzvnw5dt5ues72rk35hd69rrydxqeraz8l"

**Comment by staking-hub** [comment]
Again. Hope it is fixed.

LuckyStar

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

File: lumenx/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply lumenx/assetlist.json:3]: This would be the last blocker

Comment (staking-hub) [reply lumenx/assetlist.json]: Wow!

**Comment by staking-hub** [comment]
Thank you so much!

LuckyStar

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Pull request merged**